docs/pr-coding-guidelines.md

```diff
@@ -150,12 +150,19 @@ The js folder contains two sub folders: `lib` contains all project source files
 
 #### Linting
 
-For the consistency of our codebase, we use eslint with these rules: https://github.com/webcompat/webcompat.com/blob/master/.eslintrc. We also use Prettier for formatting our code. We have set up two recipes to check and fixe the codebase.
+To make it easier for everyone to contribute, we use [eslint](http://eslint.org/) to keep the code base steady. You can view the rules in our [ESLint config](https://github.com/webcompat/webcompat.com/blob/master/.eslintrc#L34). For further explanation of the rules, please have a look at the [ESLint documentation](http://eslint.org/docs/rules/).
 
-`npm run lint` check all JavaScript files and display the resulting errors. If you get errors, you have two possibilities. You can correct yourself or you can run `npm run fix` which use Prettier to format the code. 
+For the linting process, we use [Prettier](http://jlongster.com/A-Prettier-Formatter).
 
 In order to avoid errors during a Pull Request, `npm run lint` will be executed before each commit.
 
+`npm run lint` checks all relevant JavaScript files and displays, if something needs to be fixed.
+
+If you get an error displayed, there are two ways to fix it.
+1. You can run `npm run fix` automatically, which is great for small issues like missing spaces or lines in various files.
+2. You can correct it manually as every error message includes the file and line of the error as well as the rule which was violated.
```

Comment:
"which was violated."-> "which was violated will be displayed."

(Sorry for probably suggesting a wrong fix)